server/accounts.go

```diff
@@ -114,10 +114,10 @@ type serviceRespEntry struct {
 	msub string
 }
 
-// ServiceRespType represents the types of service request response types.
+// ServiceRespType represents the types of service req types.
 type ServiceRespType uint8
 
-// Service response types. Defaults to a singelton.
+// Servs. Defaults to a singelton.
```

Comment:
Revert.

Reply:
not even sure where that came from - find and replace maybe